app/models/webauthn_configuration.rb

```diff
@@ -4,4 +4,9 @@ class WebauthnConfiguration < ApplicationRecord
   validates :name, presence: true
   validates :credential_id, presence: true
   validates :credential_public_key, presence: true
+
+  # :reek:UtilityFunction
+  def mfa_enabled?
+    FeatureManagement.webauthn_enabled?
```

Comment:
Is this the correct logic? Why isn't there an `mfa_enabled` column in the webauthn_configurations table?

Reply:
There isn't a column because it can be simply deleted to disable it. So this will eventually be `true` when we have webauthn always on. The method is to make this configuration object the same as others so we can just run through and always as `mfa_enabled?` on any mfa configuration object.